src/Argu/Types.fs

```diff
@@ -99,10 +99,19 @@ type ArgumentType =
     /// Argument specifies a subcommand
     | SubCommand = 4
 
+/// Representation of whether spaced arguments should be tolerated
+[<RequireQualifiedAccess>]
+type SpacedArgumentsToleration =
```

Comment:
Nit: I don't see why this can't be just a bool. If a new possible state becomes necessary we can change accordingly.

Reply:
Sure thing - IMO makes the call to `validateCustomAssignmentAttributes` a little clearer, personal style on that one.